adxl345/example/examples.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2023 The Periph Authors. All rights reserved.
```

Comment:
I just realized you had put the example in a separate directory. This is not customary in Go. Any specific reason for this?

Reply:
No valuable reason :)